### PR TITLE
fix: Bug in `show` function

### LINKED ISSUE #1249: Weird issue when requesting body plot
Seems like a body plot is not working for some reason... https://geometry.docs.pyansys.com/version/stable/examples/01_getting_started/05_plotter_picker.html#Plot-objects

@AlejandroFernandezLuces could you investigate?

### PR DESCRIPTION
**Pull request by AlejandroFernandezLuces**
## Description
This should fix the plotting issues in the examples, there was a small bug in `ansys-tools-visualization-interface` that caused the clipping issues in particular. I've patch released the fix.

The CI should be fine after the patch release of  `ansys-tools-visualization-interface` is uploaded to PyPI.
## Issue linked
Closes #1249 

## Checklist
- [x] I have tested my changes locally.
- [x] I have added necessary documentation or updated existing documentation.
- [x] I have followed the coding style guidelines of this project.
- [x] I have added appropriate unit tests.
- [x] I have reviewed my changes before submitting this pull request.
- [x] I have linked the issue or issues that are solved to the PR if any.
- [x] I have assigned this PR to myself.
- [x] I have added the minimum version decorator to any new backend method implemented.
- [x] I have made sure that the title of my PR follows [Conventional commits style](https://www.conventionalcommits.org/en/v1.0.0/#summary) (e.g. ``feat: extrude circle to cylinder``)

### CODE REVIEW

File: src/ansys/geometry/core/plotting/plotter.py [hunk 1]
```diff
@@ -389,4 +389,6 @@ def show(
             Keyword arguments for the plotter. Arguments depend of the backend implementation
             you are using.
         """
-        self._backend.show(object=plotting_object, screenshot=screenshot, **plotting_options)
+        if plotting_object is not None:
+            self.plot(plotting_object, **plotting_options)
+        self._backend.show(screenshot=screenshot, **plotting_options)
```
Comment (RobPasMue) [inline src/ansys/geometry/core/plotting/plotter.py:394]: I'm curious... why is this needed?

Comment (AlejandroFernandezLuces) [reply src/ansys/geometry/core/plotting/plotter.py:394]: Not needed really, this allows the following behaviour:

```python
mesh = pv.Cylinder()
pl = Plotter()
pl.show(mesh)
```

Otherwise, you can only do the following:

```python
mesh = pv.Cylinder()
pl = Plotter()
pl.plot(mesh)
pl.show()
```

I prefer to give more options to the user

Comment (RobPasMue) [reply src/ansys/geometry/core/plotting/plotter.py:394]: Oh, gotcha - works for me

**Review by RobPasMue — APPROVED** [review]
